src/main/java/org/jitsi/utils/JNIUtils.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
```

Comment:
Can we update the license header?

Reply:
Do you have a template for what they should now be?